ui/packages/tidb-dashboard-lib/src/utils/timezone.ts

```diff
@@ -0,0 +1,39 @@
+import dayjs from 'dayjs'
+import utc from 'dayjs/plugin/utc'
+
+dayjs.extend(utc)
+
+// value range: -16 ~ 16
```

Comment:
Why the timezone range is -16 ~ 16? Are there any references?

Reply:
https://day.js.org/docs/en/manipulate/utc-offset

> If the input is less than 16 and greater than -16, it will interpret your input as hours instead.